toml/src/main/kotlin/org/rust/toml/crates/local/CratesLocalIndexStartupActivity.kt

```diff
@@ -14,15 +14,15 @@ import org.rust.openapiext.isFeatureEnabled
 
 /**
  * [CratesLocalIndexService] initializer.
- * Only tries to get service instance, without any updates to its crates index.
+ * Tries to get a service instance and binds crates local index update cargo registry index repository changes.
```

Comment:
I suppose we need to update this comment since we moved binding to index repo update into service itself

Reply:
Changed it to `Tries to get and initialize a service instance and updates it if it is needed` as the initializer by itself doesn't bind anything.